src/lib/remark-plugins/remark-image-dimensions/index.ts

```diff
@@ -0,0 +1,62 @@
+import type { Image, Root } from 'mdast'
+import { Plugin, Transformer } from 'unified'
+import { visit } from 'unist-util-visit'
+import probe from 'probe-image-size'
+
+export const remarkPluginInjectImageDimensions: Plugin = (): Transformer => {
+	return async function transformer(tree: Root) {
+		const imageNodesForDimensions = []
+		visit(tree, 'image', (node: Image) => {
+			// only transform nodes that we can fetch size on
+			if (node.url.startsWith('http')) {
```

Comment:
I actually need to scope this down to just the content api urls. `process.env.ASSET_API_ENDPOINT`?